### PR TITLE
Send secure message from the chat transcript screen

### PR DESCRIPTION
**Pull request by AndriiHorishniiMOC**
Send secure message from the chat transcript screen

[MOB-1741](https://glia.atlassian.net/browse/MOB-1741)

[MOB-1741]: https://glia.atlassian.net/browse/MOB-1741?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: widgetssdk/src/main/java/com/glia/widgets/core/engagement/GliaEngagementConfigRepository.kt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.glia.widgets.core.engagement
+
+import com.glia.widgets.chat.ChatType
+
+class GliaEngagementConfigRepository {
+    var queueIds = emptyArray<String>()
+    var chatType = ChatType.LIVE_CHAT
+
+    fun clear() {
```
Comment (DavDo) [inline widgetssdk/src/main/java/com/glia/widgets/core/engagement/GliaEngagementConfigRepository.kt:9]: maybe reset() could be a better name for this function?

**Review by DavDo — APPROVED** [review]
Good job👍

File: widgetssdk/src/main/java/com/glia/widgets/core/engagement/GliaEngagementConfigRepository.kt (hunk 1 above)
Comment (AndriiHorishniiMOC) [reply widgetssdk/src/main/java/com/glia/widgets/core/engagement/GliaEngagementConfigRepository.kt:9]: You're right!

**Review by gugalo — APPROVED** [review]

**Comment by AndriiHorishniiMOC** [comment]
!squash

**Pull request merged**